config.js

```diff
@@ -671,6 +671,25 @@ var config = {
     // is not persisting the local storage inside the iframe.
     // useHostPageLocalStorage: true,
 
+    // Moved from interfaceConfig(TOOLBAR_BUTTONS).
```

Comment:
I think we have a "UI" title somewhere in this file. Can you please make sure you put it next to it?